src/Console/src/Usage/Model/OptionDefinition.php

```diff
@@ -43,18 +43,18 @@ class OptionDefinition
      */
     private $argument;
 
-    /**
-     * Option Definition
-     *
-     * @param string $shortName
-     * @param string $longName
-     * @param string $description
-     * @param string $type
-     * @param string $default
-     * @param string $argument
-     */
-    public function __construct($shortName = null, $longName = null, $description = null, $type = 'bool', $default = null, $argument = null)
-    {
+    public function __construct(
+        ?string $shortName = null,
+        ?string $longName = null,
+        ?string $description = null,
+        string $type = self::TYPE_BOOL,
+        ?OptionValue $default = null,
```

Comment:
`OptionDefinitionParser` will create a default values always, so this argument can be made mandatory and moved into the first position. Not sure if doing that should be done now or later?

Reply:
I've done it.